app/Notifications/EmptyWikibaseNotification.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Wiki;
+use Closure;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\HtmlString;
+
+/**
+ * Notification to be sent to empty Wikibase owner if the wiki stay empty longer than 30 days
+ */
+
+class EmptyWikibaseNotification extends Notification
+{
+    /**
+     * Create a notification instance.
+     *
+     * @param string $sitename
+     * @return void
+     */
+    public function __construct(string $sitename)
+    {
+        $this->sitename = $sitename;
```

Comment:
This property still needs to be defined on the class itself, i.e. `private $sitename;`